static/src/stylesheets/module/commercial/glabs/_hosted.scss

```diff
@@ -149,7 +149,38 @@ $renaultLight: #fff9e9;
     vertical-align: top;
 
     .vjs-big-play-button .vjs-control-text {
-        display: none;
+        margin-left: -40px;
+        margin-top: -68px;
+        width: 136px;
+        height: 48px;
+        text-indent: -999px;
+
+        @include mq(tablet) {
+            margin-left: -70px;
+            margin-top: -25px;
+        }
+
+        &:before {
+            content: 'Watch now';
+            background-color: unset;
```

Comment:
It works fine at least in Chrome and Firefox. But I don't know why it's not mentioned in the spec (https://www.w3.org/TR/CSS2/colors.html#q14.0). The MDN (https://developer.mozilla.org/en-US/docs/Web/CSS/background-color) says that the initial value is `transparent` so maybe it would be safer to use this one.